internal/ycsb/ycsb.go

```diff
@@ -85,3 +109,51 @@ func Run(ctx context.Context, dir string, args []string) (*config.TestResults, e
 
 	return res, nil
 }
+
+// parseMeasurements parses go-ycsb results.
+func parseMeasurements(output string) map[string]Measurements {
+	res := make(map[string]Measurements)
+
+	scanner := bufio.NewScanner(strings.NewReader(output))
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		prefix := strings.Fields(line)[0]
```

Comment:
Do we need a condition to check for empty newline or is that impossible?
```
if len(strings.Fields(line)) == 0 {...}
```

Reply:
I haven't seen empty lines in the data, but let's add this check just in case, yeah.